packages/app/src/offscreen/Offscreen.ts

```diff
@@ -33,19 +38,25 @@ export class OffscreenController {
     merkleProofArtifacts,
     signal,
     zkeyFilePath,
-  }: SemaphoreProofRequest): Promise<FullProof> => {
-    const identityGenerated = ZkIdentitySemaphore.genFromSerialized(identitySerialized);
+  }: ISemaphoreProofRequest): Promise<FullProof> => {
+    const identity = ZkIdentitySemaphore.genFromSerialized(identitySerialized);
 
-    const identityCommitment = identityGenerated.genIdentityCommitment();
+    const identityCommitment = identity.genIdentityCommitment();
 
     const merkleProof = await getMerkleProof({
       identityCommitment,
       merkleProofArtifacts,
     });
 
-    return generateProof(identityGenerated.zkIdentity, merkleProof, externalNullifier, signal, {
+    return generateProof(identity.zkIdentity, merkleProof, externalNullifier, signal, {
       wasmFilePath: circuitFilePath,
       zkeyFilePath,
     });
   };
+
+  generateRlnProof = async (rlnProofRequest: IRlnProofRequest): Promise<string> => {
```

Comment:
We got rid of methods from cryptkeeper main handler before. I think it makes sense to keep the same agreement for this controller, too.